panels/templates/attractions/events.html

```diff
@@ -75,6 +75,7 @@
   $(function() {
     var $modals = $('.modal'),
         $signupModal = $('#signup_modal'),
+        $altSignupModal = $('#alt_signup_modal'),
```

Comment:
I liked @RobRuana 's suggestion of "alt" signup rather than "early" signup, so I renamed this to do that.